docs/astria.md

```diff
@@ -0,0 +1,145 @@
+The Astria Shared Sequencer Network replaces centralized sequencers, allowing many rollups to share a single decentralized network of sequencers that’s simple and permissionless to join. This shared sequencer network provides out-of-the-box censorship resistance, fast block confirmations, and atomic cross-rollup composability – all while retaining each rollup’s sovereignty.
+
+The Astria dev-cluster is the collective stack of all of Astria's components packaged together using Kubernetes. Although we refer to Astria as just a shared sequencer, it is composed of many smaller components that each fulfill a specific function to enable the final shared sequencing functionality. Instead of users and developers having to keep track of and manage all the individual components, the dev-cluster is built to make locally deploying and testing Astria as simple and robust as possible. 
+
+This document outlines what each of the components do, and provides some basic instructions for running the dev cluster yourself.
+
+## Architecture
+If you have seen any of [Josh's](https://twitter.com/Jskybowen) recent presentations about Astria, you will likely have seen this image:
+![Astria at a high level](images/astria-high-level.png)
+This shows the basic structure of what a shared sequencer is but purposely doesn't dive into any of the details of Astria to keep things simple. However, Astria's general architecture does break down into the basic items shown above.
+
+When expanded to show more detail, Astria's architecture looks like this:
+![Astria Architecture](images/astria-architecture.png)
+Given the diagram above, we can take a look at each of the components that make up Astria in more detail.
+
+### Users
+One of the goals of Astria is to improve the current user experience by having fast finality, Cross Rollup Composability, decentralization, and  sovereignty all in one place and out of the box. There are no hard requirements for rollups to change how their current users interact with what they offer, but additional RPCs would need to be implemented by the rollup in order for their users to take advantage of some of those features.
+
+### Rollups
+Astria is designed for permissionless rollup integration and takes advantage of lazy shared sequencing. This means that rollup developers have total sovereignty over their own execution state and can swap out sequencing layers without fear of being locked in. See the [Introduction to Astria](https://blog.astria.org/introducing-astria/) blog post for more details.
+
+When developing a rollup that utilizes Astria, there are only two interfaces that you need to satisfy:
+1. Rollup <-> Composer: 
+    - For transaction submissions.
+2. Rollup <-> Conductor: 
+    - For receiving data from the sequencing network.
+
+The [Composer](#composer) and [Conductor](#conductor) are explained in more detail in future sections.
+
+![Astria Rollup](images/astria-rollups.png)
```

Comment:
I don't like how cramped this diagram feels. Would prefer more verticality where composer/conductor are below the rollup and the arrows are straight lines up/down